dll/win32/lpk/lpk.c

```diff
@@ -63,9 +63,6 @@ LpkExtTextOut(
     const INT *lpDx,
     INT unknown)
 {
-    LPWORD glyphs = NULL;
```

Comment:
Not sure this is a change for the better. Our coding style generally prefers variables at the top of the function.